phase1/vsphere/configure-vm.sh

```diff
@@ -37,15 +61,55 @@ echo "${nodes_dns_mappings}" >> /etc/hosts
 
 
 systemctl enable docker
+if [ $? -ne 0 ]; then
+    echo "Failed to enable docker"
+    exit 1
+fi
 systemctl start docker
+if [ $? -ne 0 ]; then
+    echo "Failed to start docker"
+    exit 1
+fi
 docker run \
   --net=host \
   -v /:/mnt/root \
   -v /run:/run \
   -v /etc/kubernetes:/etc/kubernetes \
   -v /var/lib/ignition:/usr/share/oem \
   "${installer_container}" /bin/do_role
+if [ $? -ne 0 ]; then
+    echo "Failed to docker run installer container"
+    exit 1
+fi
 systemctl daemon-reload
+if [ $? -ne 0 ]; then
+    echo "Failed to reload daemon"
+    exit 1
+fi
+docker pull \
+  ${docker_registry}/hyperkube-amd64:${kubernetes_version}
+if [ $? -ne 0 ]; then
+    echo "Failed to docker pull hyperkube"
+    exit 1
+fi
 systemctl enable kubelet
+if [ $? -ne 0 ]; then
+    echo "Failed to enable kubelet"
+    exit 1
+fi
 systemctl start kubelet
+if [ $? -ne 0 ]; then
+    echo "Failed to start kubelet"
+    exit 1
+fi
+
+if [ "`systemctl is-active kubelet`" != "active" ]; then
+    echo "Failed to detect kubelet service active"
+    exit 1
+fi
 
+if [ "${role}" == "master" ]; then
```

Comment:
This code was removed recently. Do you plan to reintroduce it?
https://github.com/kubernetes/kubernetes-anywhere/commit/81315507b545306a18874547c289d7d626ff72eb#diff-9a59af74f3d758c7db22a7d5e0a315b7L52